### PR TITLE
add more clarity to wofs groups

### PR DESCRIPTION
**Pull request by pindge**
- add clarity to wofs grouped layers
- increase wofs new style legend size

### CODE REVIEW

File: dev/services/wms/ows_refactored/wofs/ows_wofs_annual_cfg.py [hunk 1]
```diff
@@ -166,7 +166,7 @@
 
 
 layers = {
-    "title": "Water Observations from Space",
+    "title": "Water Observations from Space annual statistics",
```
Comment (alexgleith) [inline dev/services/wms/ows_refactored/wofs/ows_wofs_annual_cfg.py:169]: Please keep consistent case

File: dev/services/wms/ows_refactored/wofs/ows_wofs_seasonal_cfg.py [hunk 2]
```diff
@@ -160,7 +160,7 @@
 
 
 layers = {
-    "title": "Water Observations from Space",
+    "title": "Water Observations from Space Seasonal",
```
Comment (alexgleith) [inline dev/services/wms/ows_refactored/wofs/ows_wofs_seasonal_cfg.py:163]: This is Capital

File: dev/services/wms/ows_refactored/wofs/ows_wofs_summary_cfg.py [hunk 3]
```diff
@@ -182,7 +182,7 @@
 
 
 layers = {
-    "title": "Water Observations from Space",
+    "title": "Water Observations from Space summary statistics",
```
Comment (alexgleith) [inline dev/services/wms/ows_refactored/wofs/ows_wofs_summary_cfg.py:185]: This is not

**Review by alexgleith — CHANGES_REQUESTED** [review]

File: dev/services/wms/ows_refactored/wofs/ows_wofs_seasonal_cfg.py (hunk 2 above)
Comment (pindge) [reply dev/services/wms/ows_refactored/wofs/ows_wofs_seasonal_cfg.py:163]: @erin-telfer , is this `Seasonal Statistics` or just `Seasonal`, asking for clarification ,

File: dev/services/wms/ows_refactored/wofs/ows_wofs_annual_cfg.py (hunk 1 above)
Comment (erin-telfer) [reply dev/services/wms/ows_refactored/wofs/ows_wofs_annual_cfg.py:169]: "title": "Water Observations from Space Annual Statistics",

File: dev/services/wms/ows_refactored/wofs/ows_wofs_seasonal_cfg.py (hunk 2 above)
Comment (erin-telfer) [reply dev/services/wms/ows_refactored/wofs/ows_wofs_seasonal_cfg.py:163]: It shouldn't be either.

**Review by SpacemanPaul — APPROVED** [review]
Looks fine to me.

**Pull request merged**